contracts/modules/staking/StakingProducts.sol

```diff
@@ -599,8 +605,75 @@ contract StakingProducts is IStakingProducts, MasterAwareV2, Multicall {
 
   /* dependencies */
 
+  function tokenController() internal view returns (ITokenController) {
+    return ITokenController(internalContracts[uint(ID.TC)]);
+  }
+
+  function cover() internal view returns (ICover) {
+    return ICover(coverContract);
+  }
+
+  function coverProducts() internal view returns (ICoverProducts) {
+    return ICoverProducts(getInternalContractAddress(ID.CP));
+  }
+
   function changeDependentContractAddress() external {
     // none :)
   }
 
+  function createStakingPool(
```

Comment:
i can't recall what was the latest discussion on this function

happy to move it to CoverProducts if it makes sense.

Reply:
There can be only one operator so it cannot live in more than one contract. See other comment regarding the proposal to move it back to `Cover`.